src/render.c

```diff
@@ -452,11 +452,18 @@ static void render_con_stacked(Con *con, Con *child, render_params *p, int i) {
     child->rect.height = p->rect.height;
 
     child->deco_rect.x = p->x - con->rect.x;
-    child->deco_rect.y = p->y - con->rect.y + (i * p->deco_height);
     child->deco_rect.width = child->rect.width;
-    child->deco_rect.height = p->deco_height;
 
-    if (p->children > 1 || (child->border_style != BS_PIXEL && child->border_style != BS_NONE)) {
+    if (config.force_tabbed_stacked_titlebar) {
+        child->deco_rect.y = p->y - con->rect.y + (i * p->deco_height);
+        child->deco_rect.height = p->deco_height;
+    } else {
+        child->deco_rect.y = p->y - con->rect.y;
+        child->deco_rect.height = (child->border_style == BS_PIXEL ? 1 : 0);
```

Comment:
Where does this `1` come from? Actually, this makes me think… shouldn't the descendants of stacked / tabbed containers also get the border on the top if `force_tabbed_stacked_titlebar` is disabled?